src/courses/advanced/05.md

```diff
@@ -42,28 +42,51 @@ To write resources, we first need to know what we are testing! In your Codespace
 
-Now let's write some tests and confirm that they run. You can put these tests in the `example.rb` file generated in the `controls` folder of your `my_git` InSpec profile. These tests are written using the `command` resource which is provided by InSpec. We will write a `git` resource in this section to improve this test. **Note that you will need to put the full directory path of the `.git` file from your `git_test_target` repository as the `git_dir` value on line 4 of `example.rb`. To get the full path of your current location in the terminal, use `pwd`.**
+Now let's write some tests and confirm that they run. You can put these tests in the `example.rb` file generated in the `controls` folder of your `my_git` InSpec profile. These tests are written using the `command` resource which is provided by InSpec. We will write a `git` resource in this section to improve this test.
 
-```ruby
-# encoding: utf-8
-# copyright: 2018, The Authors
+::: note Inputs review
+The `git_dir` input will be used in our profile.  It needs to contain the path of the `.git` folder you are assessing.  Usually, you'd want to put a default location for that input in your `inspec.yml`, such as `./.git`, that the user of your profile could overwrite with an inputfile, but for expediency's sake we're going to hardcode it to the `.git` folder inside of your `git_test_target` repository. The value in the codeblock below should be correct, but it's always good to doublecheck by using the `cd` command to navigate inside of the `.git` folder in the `git_test_target` repository and then using the `pwd` command to get the full path.
+:::
+
+::: code-tabs
+
+@tab inspec.yml
+```yaml
+name: my_git
+title: InSpec Profile
+maintainer: The Authors
+copyright: The Authors
+copyright_email: you@example.com
+license: Apache-2.0
+summary: An InSpec Compliance Profile
+version: 0.1.0
+supports:
+  platform: os
+
+inputs:
+  - name: git_dir
+    type: String
+    value: /workspaces/saf-training-lab-environment/git_test_target/.git
+```
 
-git_dir = "/workspaces/saf-training-lab-environment/git_test_target_target/.git"
+@tab example.rb
+```ruby
+git_dir = input('git_dir')
 
 # The following branches should exist
-describe command("git --git-dir #{git_dir} branch") do
-  its('stdout') { should match /main/ }
+describe command("git --git-dir #{git_dir} branch --format='%(refname:short)'").stdout.lines.map(&:strip) do
+  it { should include 'main' }
 end
 
-describe command("git --git-dir #{git_dir} branch") do
-  its('stdout') { should match /testBranch/ }
+describe command("git --git-dir #{git_dir} branch --format='%(refname:short)'").stdout.lines.map(&:strip) do
+  it { should include 'testBranch' }
 end
 
 # What is the current branch
-describe command("git --git-dir #{git_dir} branch") do
-  its('stdout') { should match /^\* main/ }
+describe command("git --git-dir #{git_dir} branch --show-current").stdout.strip do
+  it { should match 'main' }
```

Comment:
While we're at it, this matcher should be `eq` and not `match`. We're not matching this as a regex at this point and while this technically works we want to reflect a good habit of using the correct matcher for the situation.

Reply:
done